apps/remix-ide/src/app/tabs/compileTab/compilerContainer.js

```diff
@@ -499,18 +502,20 @@ class CompilerContainer {
 
   // fetching both normal and wasm builds and creating a [version, baseUrl] map
   async fetchAllVersion (callback) {
-    let allVersions, selectedVersion, allVersionsWasm, isURL
+    let selectedVersion, allVersionsWasm, isURL
+    let allVersions = [{ path: 'builtin', longVersion: 'latest local version' }]
```

Comment:
Can we put a exact version here? "Latest local (v0.10.12)"

Reply:
we don't know before hand which version will be loaded cause it is the one located in the asset folder, we could hardcode it, we'll need to change it each time we put a new version in the asset